Packs/CheckpointFirewall/Integrations/CheckPointFirewallV2/CheckPointFirewallV2.py

```diff
@@ -1796,6 +1802,8 @@ def main():
     password = params.get('username', {}).get('password')
 
     server = params['server']
+    if server[-1] == "/":
```

Comment:
Remind my why did we add this?

Reply:
If the user enters in the URL field something like this **www.example.com/**, Later in the code we connect the port to the url so it's will be like this **www.example.com/:443/web_api** instead of **www.example.com:443/web_api**
<img width="989" alt="Screen Shot 2021-06-10 at 16 03 37" src="https://user-images.githubusercontent.com/78307768/121530210-dd37c900-ca05-11eb-9cb2-16e1b5ae5963.png">